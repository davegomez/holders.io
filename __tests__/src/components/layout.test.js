import React from 'react'
import Layout from '../../../src/components/layout'
import renderer from 'react-test-renderer'

test('ThemeSelector component', () => {
  const component = renderer.create(
    <Layout url='foo' />
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
