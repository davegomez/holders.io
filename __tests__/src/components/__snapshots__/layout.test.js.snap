// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemeSelector component 1`] = `
<div
  data-jsx={630023836}
  style={
    Object {
      "position": "relative",
    }
  }
>
  <div
    className="navbar"
    data-jsx={2687871026}
  >
    <ul
      data-jsx={1218288577}
    >
      <li
        className="active"
        data-jsx={1218288577}
      >
        <a
          data-jsx={1218288577}
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="active"
        data-jsx={1218288577}
      >
        <a
          data-jsx={1218288577}
          href="/docs"
          onClick={[Function]}
        >
          Documentation
        </a>
      </li>
      <li
        className="active"
        data-jsx={1218288577}
      >
        <a
          data-jsx={1218288577}
          href="/about"
          onClick={[Function]}
        >
          About
        </a>
      </li>
    </ul>
    <a
      aria-label="Fork me on GitHub"
      data-jsx={1144686127}
      href="https://github.com/davegomez/holders.io"
    >
      <svg
        data-jsx={1144686127}
        height="30"
        width="31"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M15.378 0c8.49 0 15.378 6.888 15.378 15.378 0 6.788-4.405 12.555-10.512 14.597-.78.14-1.062-.34-1.062-.74 0-.5.02-2.163.02-4.226 0-1.442-.48-2.363-1.04-2.844 3.423-.38 7.028-1.682 7.028-7.59 0-1.68-.6-3.042-1.582-4.124.16-.4.68-1.962-.16-4.085-1.282-.4-4.225 1.582-4.225 1.582-1.222-.34-2.543-.52-3.845-.52s-2.623.18-3.844.52c0 0-2.944-1.983-4.225-1.583-.842 2.123-.322 3.685-.162 4.085-.98 1.082-1.58 2.443-1.58 4.125 0 5.887 3.583 7.21 7.007 7.59-.44.4-.84 1.08-.98 2.06-.882.402-3.125 1.083-4.467-1.28-.84-1.462-2.362-1.582-2.362-1.582-1.502-.02-.1.94-.1.94 1 .462 1.7 2.244 1.7 2.244.903 2.742 5.188 1.82 5.188 1.82 0 1.282.02 2.484.02 2.864 0 .4-.28.88-1.062.74C4.405 27.935 0 22.167 0 15.38 0 6.888 6.888 0 15.378 0zm-9.55 22.086c-.04.08-.16.1-.26.04-.122-.06-.182-.16-.142-.24.04-.06.14-.08.26-.04.12.06.18.16.14.24zm.62.68c-.08.08-.22.04-.32-.06-.1-.12-.12-.26-.04-.32.08-.08.22-.04.32.06.1.12.12.26.04.32zm.6.902c-.08.06-.24 0-.34-.14-.1-.14-.1-.3 0-.36.1-.08.26-.02.34.12.1.14.1.3 0 .38zm.84.84c-.08.1-.26.08-.4-.06-.14-.12-.18-.3-.08-.38.08-.1.26-.08.4.06.12.12.16.3.08.38zm1.143.5c-.04.12-.22.182-.38.12-.18-.04-.3-.18-.26-.3.04-.12.22-.18.38-.14.18.06.3.2.26.32zm1.262.1c0 .122-.14.222-.32.222-.2.02-.34-.08-.34-.22 0-.12.14-.22.32-.22.18-.02.34.08.34.22zm1.16-.2c.02.12-.1.24-.28.282-.18.04-.34-.04-.36-.16-.02-.14.1-.26.28-.3.18-.02.34.06.36.18z"
          data-jsx={1144686127}
          fill="#333"
          fillRule="evenodd"
        />
      </svg>
    </a>
  </div>
  <div
    className="container"
    data-jsx={630023836}
  />
  <footer
    data-jsx={630023836}
  >
    <p
      data-jsx={630023836}
    >
      Powered by 
      <a
        aria-label="Powered by Now"
        data-jsx={630023836}
        href="https://zeit.co/now"
      >
        <svg
          data-jsx={2430622347}
          viewBox="0 0 40 35"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            data-jsx={2430622347}
            fill="none"
            fillRule="evenodd"
            stroke="rgba(0, 0, 0, 1)"
          >
            <path
              d="M19.844 2.047l17.845 31.95H2z"
              data-jsx={2430622347}
              strokeWidth="2"
            />
            <path
              d="M36.085 33.59L19.235 3.23M36.85 33.59L20 3.23M35.32 33.59L19.233 4.05M34.553 33.59L19.233 4.872M18.468 4.872l16.85 28.718M33.787 33.59L18.467 5.692M30.723 27.026L24.596 16.36"
              data-jsx={2430622347}
              strokeLinecap="square"
            />
          </g>
        </svg>
         Now
      </a>
    </p>
  </footer>
</div>
`;
